rxnorm/rxnorm_builder.py

```diff
@@ -102,14 +104,25 @@ def parse_ctl_file(self, contents: list[str]) -> (str, dict):
                 continue
             if line.startswith("infile"):
                 datasource = line.split(" ")[1].rstrip().replace("'", "")
+            elif line == "trailing nullcols(\n":
+                is_col_def_section = True
+                continue
             elif line.startswith("("):
                 is_col_def_section = True
                 line = line[1:]
-            elif line.startswith(")"):
+            elif line.strip(" ").startswith(")"):
```

Comment:
This makes me believe that the .ctl table configs are hand managed, which makes this a bit more of a headache. While specific to rxnorm, this could be shared elsewhere for safety.